_posts/2022-06-08-cockpit-271.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Cockpit 271
+author: garrett
+date: '2022-06-08'
+tags: cockpit
+slug: cockpit-271
+category: release
+summary: 'Container names on the metrics page'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit-podman 49.1:
+
+
+## Show container names on the Metrics page
+
+Instead of just showing a container ID, Cockpit now displays Podman container names in the CPU and Memory usage cards on the "Metrics and history" page.
+
+Additionally, after clicking a container name on the Metrics page, you'll see just the container you clicked on within Cockpit-Podman, thanks to improved filtering.
+
+![screenshot of ## show container names on the metrics page]({{ site.baseurl }}/images/271-show-container-names-on-the-metrics-page.png)
+
+
+## Try it out
+
+Cockpit 271, cockpit-machines 270.1 and cockpit-podman 49.1 is available now:
+
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/271)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-4869d82c66)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/FEDORA-2022-80d2aa8e21)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/270.1)
+* [cockpit-machines Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-d30c57494e)
+* [cockpit-machines Fedora 35](https://bodhi.fedoraproject.org/updates/FEDORA-2022-f584409196)
+* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/49.1)
+* [cockpit-podman Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-dcc6dd5d17)
+* [cockpit-podman Fedora 35](https://bodhi.fedoraproject.org/updates/FEDORA-2022-f91d6f8df9)
```

Comment:
These are the obsolete updates for the 49/270 versions, replaced with .1 versions.